weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -597,13 +597,13 @@ function makeCodeText(
   }
   if (Object.values(filter).some(value => value !== undefined)) {
     codeStr += `    filter={`;
-    if (filter.opVersionRefs) {
+    if (filter.opVersionRefs && filter.opVersionRefs.length > 0) {
```

Comment:
Should we be more explicit than truthiness? E.g.
```suggestion
    if (filter.opVersionRefs !== undefined && filter.opVersionRefs.length > 0) {
```

same below.

Reply:
i mean, its typescript, we know its going to be either undefined or a list. If this were python it would already be working, but truthiness of typescript evals empty list is True (instead of False in python).